### ISSUE TITLE
create daily flows report

### ISSUE THREAD

**Issue by dormerod** [bugflow/kanned-bananas#3]
See also: bugflow/taiga-bugflow/issues/8

```
as a delivery manager or product owner
I need a daily report summarising the kanban flows since yesterday
so that I can track spending and demonstrate progress to clients
```

**Comment by dormerod** [comment]
similar to #2, but (potentially a subset of) this is the team level version of "what I did yesterday".

**Comment by dormerod** [comment]
This is working well and occasional issues with duplicates were fixed in #10.
